crates/hyperqueue/src/client/commands/submit/command.rs

```diff
@@ -687,6 +687,10 @@ pub struct JobResubmitOpts {
     /// Job that should be resubmitted
     job_id: u32,
 
+    /// Keep task(s) that were present at crash of the worker
+    #[clap(long)]
+    keep_problematic: bool,
```

Comment:
Please remove the `resubmit` changes from this PR and focus this PR only on cancelling the tasks, that's the main objective.